docs/workforce-management-framework/topics/con-KeycloakVsPassport.adoc

```diff
@@ -0,0 +1,37 @@
+[]
+= Introduction to {WFM-RC-NameShort} security features
+
+{WFM-RC-NameShort} currently supports two security implementations
+Keycloak and Passport.js.
+
+== Passport.js
+Passport.js is a lightweight authentication middleware for Node.js that is Extremely flexible and modular.
+A comprehensive set of strategies support authentication using a username and password, Facebook, Twitter, and more.
```

Comment:
Do we expect the reader to know what is a 'strategy' is or should we describe it?